### ISSUE TITLE
VPN Problems

### ISSUE THREAD

**Issue by Electricity2015** [Ylianst/MeshCentral#2068]
After a user is conecting to VPN, I can no longer connect to the device.

**Comment by krayon007** [comment]
What is the network topology after the VPN is connected? Is a proxy required for internet access? I know with most of the corporate VPN's I've used, the routing tables are altered upon connection to force all traffic thru the VPN. And if there is a proxy, how is it configured/setup? Normally the agent tries to detect the proxy using a couple different methods, altho it doesn't support autoconfiguration scripts. If you manually configure a proxy in the agent's msh, the agent should automatically try to use it when necessary.

Also what OS is the agent running?

**Comment by Electricity2015** [comment]
yes, is forcing all trafic trought vpn ,the vpn type is L2TP/IPsec with pre-shared key , it is vpn from Meraki.
The external ip adress is changing to.

**Comment by krayon007** [comment]
When your VPN is connected, is a proxy required to reach the mesh server?

**Comment by Electricity2015** [comment]
No,there is no proxy.
